src/components/microlearning/styles/VideoDetailPage.scss

```diff
@@ -81,4 +81,32 @@
   .vjs-transcribe-cueline {
     color: #00688d;
   }
+  .logo-cutom-style {
+    max-width: 59px;
+    max-height: 30px;
+    border-radius: 4px;
+    box-shadow: 0 0 8px 0 rgba(0, 0, 0, 0.15), 0px 0px 4px 0px rgba(0, 0, 0, 0.15);
+  }
+}
+.preview-enroll-expand-body {
+  max-height: 120px;
+  height: auto;
+  overflow: hidden;
+  position: relative;
+  -webkit-transition: max-height .5s ease;
+  transition: max-height .5s ease;
+
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 24px;
+    background-image: -webkit-gradient(linear,left top,left bottom,from(hsla(0,0%,100%,0)),to(white));
+    background-image: linear-gradient(180deg,hsla(0,0%,100%,0),white);
+  }
+  ul {
+    padding-left: 1.5rem;
+  }
 }
```

Comment:
Since you made changes to the `VideoDetailPage.scss`, I recommend testing the page with video data to ensure that the CSS related to the video player is not affected. The screenshot you provided does not include the video player..

Reply:
i tested it with video data too and it's working fine.